### DISCUSSION TITLE
[clues suggestion] Make fairy ring scroll to right one

### DISCUSSION THREAD

**Discussion by Seismicwand** [runelite/runelite#15081] [Ideas]
When doing a clue, if it requires a fairy ring, could you make the fairy ring interface automatically scroll to the right fairy ring code?

This may be difficult as you need it unlocked etc and I understand if its not possible due to this, but like with the music tracks and Cecilia and emote clues, I think it would be a nice addition :)

**Comment by Nightfirecat** [comment]
We can't fake inputs to the fairy ring code wheels, that would be botting.

**Reply by pwatts6060** [reply]
Even just the scroll bar on the interface? They don't mean actually entering AKS etc into the interface, just positioning the scroll in the list panel to have the right option in view without needing to scroll more.

**Reply by Nightfirecat** [reply]
That could be doable.

**Reply by Seismicwand** [reply]
Yes what Pwatts said, just scroll to the right one so we can click on it, maybe highlight it like emotes are highlighted? 
Thanks :)

**Reply by alexkingcodes** [reply]
The Hunter Rumours plugin does this, would be nice if Clue Scrolls did too.
